pkg/addonmanager/addon.go

```diff
@@ -49,14 +52,22 @@ func init() {
 }
 
 type subscriptionAgent struct {
-	agentImage string
-	kubeClient kubernetes.Interface
+	agentImage           string
+	kubeClient           kubernetes.Interface
+	agentInstallStrategy *agent.InstallStrategy
 }
 
-func NewAgent(agentImage string, kubeClient kubernetes.Interface) agent.AgentAddon {
+func NewAgent(agentImage string, kubeClient kubernetes.Interface, agentInstallAllStrategy bool) agent.AgentAddon {
+	var agentInstallStrategy *agent.InstallStrategy
+	agentInstallStrategy = nil
+	if agentInstallAllStrategy == true {
```

Comment:
I think the lint wants a space in between 
```
agentInstallStrategy = nil

	if agentInstallAllStrategy == true {
```

you should be able to run `make lint` locally and see.